Lib/AspNet/Microsoft.WindowsAzure.Storage/project.json

```diff
@@ -1,91 +1,74 @@
 ﻿{
-    "version": "7.0.1.0",
+    "version": "7.1.1.0",
     "authors": [ "Microsoft Corporation" ],
     "description": "Azure Storage SDK for CoreCLR Preview",
     "dependencies": {
         "Microsoft.Data.OData": "5.6.4",
```

Comment:
@erezvani1529 I see what you wrote for my line comments, but I think this is part of a larger issue here.

Allow me to vent a little. I'm getting a little peeved again. [It's a proverbial @ronnyek (@ron_frown) moment! :smile:] These packages (e.g., `Microsoft.Data.OData`) are *still* not upgraded for .NET Core. I realize this is internal, political, and technical (i.e., "hard to do") stuff. I also realize that these packages are provided by another team, so I know you can't really make them go faster. I can't close this contract I'm working on with preview packages in it, and it's taking *forever*  to get everything to RTM/GA (Nano, included).

The OData people have had *an eternity* to get their work done. I asked them at https://github.com/OData/odata.net/issues/620 about what's going on, but I fear that they won't answer. Just look at this lack of responsiveness ... https://github.com/OData/odata.net/issues ... not good ... not good.

You should have known about `Newtonsoft.Json 9.0.1`, and it should have been addressed in this release. I don't quite make the issue connection on `netcoreapp1.0` TFM (I'm not going to drill down on understanding it ... I've got to get back to work on a project here), but that's another one that should have been known/determined for this release. I mean you have to ask yourself in the code review, "Is this an app?" "Why is this here?" I don't understand why your devs didn't add a line comment in the source if it was a workaround. Anyway ... doesn't matter. My point is just that .NET Core still doesn't seem to be receiving priority attention.

@niemyjski asked in https://github.com/Azure/azure-storage-net/issues/298 what's blocking this lib ... no answer. It sort of piles on, because if you see for the second time that an old package was included in a new preview (+ the bad TFM) and then your prior experience was https://github.com/Azure/azure-storage-net/issues/230#issuecomment-184910568, you cringed when you saw the response in https://github.com/Azure/azure-storage-net/issues/266, and you saw the poor decision making (IMO) in https://github.com/Azure/azure-storage-net/issues/173 ... you get a building impression, and it's not a good one.

Your ReadMe is not helpful wrt roadmap, planning, breaking issues, and releases. The changelog is buried and not something we can't subscribe to and only updated after-the-fact. The team blog is too general and infrequently updated and never speaks to planning, roadmap, and WIP. As for known issues, I don't even know where that is! I wish you would adopt more of an "open an issue first ... discuss the issue there ... start the PR separately ... discuss the PR there ... then :shipit: ... then merge" approach ... just like they do over in the .NET Core/CoreFX GH repos. There is a much better chance that community :cat2:s like @bartmax and myself can jump in and ask a question or point out a potential problem or ask @davidfowl to check something that looks :fish:y.

The community can only hope that communication with @davidfowl firms up across teams ... not just here ... all teams that touch on .NET Core.

With the integration of services required in the modern .NET app, it's not really fair to say that .NET Core is "RTM" when significant dependencies for apps (e.g., Azure Storage lib) are still in preview/pre-release. Having to go with REST API here is not a feasible workaround (given what I was paid by the hotel) and carries its own dangers (easier to make a fatal mistake in development).

Ok ... rant over. Thanks for hearing me out. :smile: ... I am just trying to get The Shoreham Hotel contract wrapped up (a year after starting with .NET Core), and this is taking *toooooo long* at this point and there are clearly some communication problems that even a lowly green dinosaur can see are not being addressed.

cc/ @bartmax @shanselman @jongalloway @damianedwards @coolcsh